src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -869,7 +947,7 @@ def resolve_imports_and_packages(
                     "or a tuple of the file path (str) and the import path (str)."
                 )
             udf_level_imports[resolved_import_tuple[0]] = resolved_import_tuple[1:]
-        all_urls = session._resolve_imports(
+        all_urls = session._resolve_imports(  # _resolve_imports will return an empty list in case of sandbox
```

Comment:
For offline discussion, is it possible to skip using session if the native apps sandbox doesn't have a session?